config/tags.php

```diff
@@ -206,8 +206,8 @@
 
 			// if value is a UUID, resolve to page/file model
 			// and use the URL as value
-			if (Uuid::is($tag->value, ['page', 'file']) === true) {
-				$tag->value = Uuid::for($tag->value)?->toUrl();
+			if ($uuid = Uuid::from($tag->value, ['page', 'file'])) {
+				$tag->value = $uuid->toUrl();
```

Comment:
`Issue`: This changes behavior. Before, a valid but non-existent UUID would lead to an empty value, which would be handled below. Now, the original UUID URI would stay in the value unconverted.

Reply:
I think here we could solve it by just turning it into a one-liner:

```php
$tag->value = Uuid::from($tag->value, ['page', 'file'])?->toUrl();
```

as this would also lead to `$tag->value` being `null` afterwards if it isn't a valid UUID.

Reply:
Not quite. Then the value will also be null for all non-UUID links (links that don't match the `page://something` or `file://something` patterns).

Reply:
Reverted it